### ISSUE TITLE
Alert bash users to the ramifications of relying on bash-preexec

### ISSUE THREAD

**Issue by cmm** [ellie/atuin#752]
Since Atuin deals with shell history, bash-preexec's unconditional removal of `ignorespace` from `HISTOPTS` and the potential privacy-related surprises due to that are very relevant, and should be mentioned somewhere in the `README.md` (I guess? Or maybe as an echo when eval'ing `atuin init bash`) very prominently.

**Comment by conradludgate** [comment]
We should be implementing the same considerations as ignorespace already. There should be no privacy concerns here

**Comment by ellie** [comment]
Yeah we've done this since #123, should put something in the README though. Tbh I'll get some sort of privacy section going

**Comment by ellie** [comment]
Hey! Thanks for pointing this out @cmm - totally misunderstood your prior issue and missed the reply until now 😖

I've just updated the README and the website to include a note about this. Shall see if we can change preexec 🙏
